0.4/examples/invalid/invalid_axis_units.json

```diff
@@ -0,0 +1,33 @@
+{
+    "multiscales": [
+        {
+            "axes": [
+                {
+                    "name": "y",
+                    "type": "space",
+                    "units": "micron"
```

Comment:
Same as for invalid type: this is currenrtly a SHOULD: https://github.com/ome/ngff/blob/main/latest/index.bs#L218